File: Ramsha.Ui/src/app/models/statistics/adminStatistics.ts
import { SalesPerformance } from "./statistic";


export type TotalCategoryProducts = {
    label: string;
    value: number
}

export type AdminStatistics = {
    totalCompletedOrders: number;
    totalCustomers: number;
    totalSuppliers: number;
    totalProducts: number;
    salesPerformance: SalesPerformance[];
    totalCategoriesProducts: TotalCategoryProducts[]
};

File: Ramsha.Application/Dtos/Statistics/AdminStatistics.cs


namespace Ramsha.Application.Dtos.Statistics;

public record AdminStatistics(
    int TotalCompletedOrders,
    int TotalCustomers,
    int TotalSuppliers,
    int TotalProducts,
List<SalesPerformance> SalesPerformance,
List<TotalCategoryProducts> TotalCategoriesProducts
);


public record TotalCategoryProducts(
string Label,
int Value
);

